doc/build/dts/howtos.rst

```diff
@@ -122,21 +122,18 @@ that the device is ready to be used before passing it to any API functions.
 
 In some situations the device cannot be known at build-time, e.g., if it depends
 on user input like in a shell application. In this case you can get the
-``struct device`` by combining :c:func:`DT_LABEL` with
+``struct device`` by combining :c:func:`DEVICE_DT_NAME` with
 :c:func:`device_get_binding`:
 
 .. code-block:: c
 
-   const struct device *uart_dev = device_get_binding(DT_LABEL(MY_SERIAL));
+   const struct device *uart_dev = \
+      device_get_binding(DEVICE_DT_NAME(MY_SERIAL));
 
 You can then use ``uart_dev`` with :ref:`uart_api` API functions like
 :c:func:`uart_configure`. Similar code will work for other device types; just
 make sure you use the correct API for the device.
 
-There's no need to override the ``label`` property to something else: just make
-a node identifier and pass it to ``DT_LABEL`` to get the right string to pass
-to ``device_get_binding()``.
-
```

Comment:
Looking at it again, this whole example doesn't really make sense to me. If the device name comes from user input, then we aren't getting it from a DT node at all. Just deleting this part completely makes more sense to me right now.

Reply:
Removed, but I'll leave this one open in case there are other opinions.